aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -263,6 +263,16 @@ module AptosFramework::Coin {
         Coin<CoinType> { value: amount }
     }
 
```

Comment:
If we're adding ways to create mint/burn caps, we should probably add ways to destroy them as well (they're not droppable).